packages/database/test/exp/integration.test.ts

```diff
@@ -20,13 +20,19 @@ import { initializeApp, deleteApp } from '@firebase/app-exp';
 import { expect } from 'chai';
 import { DATABASE_ADDRESS, DATABASE_URL } from '../helpers/util';
 
-import { getDatabase } from '../../exp/index';
+import {
+  getDatabase,
+  goOffline,
+  goOnline,
+  ref,
+  refFromURL
+} from '../../exp/index';
 
 export function createTestApp() {
   return initializeApp({ databaseURL: DATABASE_URL });
 }
 
-describe('Database Tests', () => {
+describe.skip('Database Tests', () => {
```

Comment:
Do you want to put a todo to make sure this is removed when the code is ready or will this be obvious enough to remember?

Reply:
Better safe than sorry :)